packages/react-magma-dom/src/components/StyledButton/styles.ts

```diff
@@ -77,23 +77,47 @@ export function buildButtonPadding(props) {
 export function buildButtonBackground(props) {
   props = buildPropsWithDefaultButtonStyles(props);
 
-  if (props.variant !== 'solid' && props.color !== 'marketing') {
-    return 'rgba(0,0,0,0)';
-  }
   if (props.disabled) {
-    return props.theme.colors.neutral06;
-  }
-  if (props.color === 'marketing') {
-    return props.theme.colors.pop04;
+    if (props.isInverse) {
+      if (props.color === 'secondary' || props.variant === 'link') {
+        return 'none';
+      }
+      return transparentize(0.7, props.theme.colors.neutral100);
+    }
+    if (props.color === 'secondary') {
+      return props.theme.colors.neutral100;
+    }
+    if (props.variant === 'link') {
+      return 'none';
+    }
```

Comment:
If it can be link && secondary, which is the expected value? Just want to make sure these two checks are in the correct order